src/pages/Encounters/tabs/EncounterNotesTab.tsx

```diff
@@ -518,7 +518,7 @@ export const EncounterNotesTab = ({
           </div>
         </div>
 
-        <ScrollArea className="flex-1">
+        <ScrollArea className="flex-1 overflow-y-auto">
```

Comment:
You do not need to add `overflow-y-auto` manually in ScrollArea right ? 

```suggestion
        <ScrollArea className="flex-1">
```

Reply:
> You do not need to add `overflow-y-auto` manually in ScrollArea right ?

It is invalid